### DISCUSSION TITLE
Support Bitbucket API@1.0.0

### DISCUSSION THREAD

**Discussion by agrouse** [renovatebot/renovate#32551] [Request Help] [answered]
### Tell us more.

Hi there,

Looking for support or ideas around supporting the Bitbucket API@1.0.0 in latest version of renovate. Would it be possible to add 'platform=bitbucket-server-legacy' so that renovate can open PR's on my enterprise's repos? We're currently using version 35.98.2 of renovate which works fine. However, we need support for fetching presets on http servers which is not possible in this version.

Thanks!
Amy

**Comment by rarkins** [comment]
Are you using one of the end of life'd Bitbucket Server versions?

**Reply by agrouse** [reply]
Yes, can we add the old bitbucket-server platform from version 35.98.2 and rename it to bitbucket-server-legacy?

**Reply by agrouse** [reply]
also, happy to make a contribution for it

**Reply by viceice** [reply]
nope, please provide more details which version of bitbucket server you are using and which failures do you get from never renovate versions.

**Reply by agrouse** [reply, marked as answer]
Sure, it's Bucket Data Center 7.21.4 (https://confluence.atlassian.com/display/BitbucketServer/Bitbucket+Data+Center+and+Server+7.21+release+notes). I think it was user error on my part--looking back it looks like I may have specified platform to 'bitbucket' instead of 'bitbucket-server'. The source code shows the correct REST calls, and it's running in my pipeline.
